src/audio-worklet/basic/hello-audio-worklet/main.js

```diff
@@ -3,24 +3,50 @@
 // found in the LICENSE file.
 
 const audioContext = new AudioContext();
+let isModuleLoaded = false;
+let isPlaying = false;
+let oscillatorNode = null;
 
 const startAudio = async (context) => {
-  await context.audioWorklet.addModule('bypass-processor.js');
-  const oscillator = new OscillatorNode(context);
-  const bypasser = new AudioWorkletNode(context, 'bypass-processor');
-  oscillator.connect(bypasser).connect(context.destination);
-  oscillator.start();
+  if (!isModuleLoaded) {
+    await context.audioWorklet.addModule('bypass-processor.js');
+    oscillatorNode = new OscillatorNode(context);
+    const bypasser = new AudioWorkletNode(context, 'bypass-processor');
+    oscillatorNode.connect(bypasser).connect(context.destination);
+    oscillatorNode.start();
```

Comment:
I think L13~L16 can be extracted to a different method.

`isModuleLoaded` should guard around the `addModule` method, nothing else.